### PR TITLE
283-readme-add-subtasks

### PR DESCRIPTION
**Pull request by jimmynotjim**
Include available grunt sub-tasks in readme for easier discovery

## Additions

- Added `grunt watch` sub-tasks to readme

## Removals

- None

## Changes

- None

## Testing

- None

## Review

- @Scotchester 
- @anselmbradford 
- @sebworks 

## Preview

None

## Notes

- `:css` and `:cssjs` sub-tasks are much faster and make working locally easier
- Fixes 283

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -182,6 +182,16 @@ open a terminal and run:
 $ grunt watch
 ```
 
+*Alternatively if you don't want to run the full watch task there are two available sub-tasks*
```
Comment (anselmbradford) [inline README.md]: Might include the ending colon like the sentence before this one. Also, I'm no grammar wiz, but I think you're missing a comma:
`Alternatively, if you don't want to run the full watch task there are two available sub-tasks:`

Comment (Scotchester) [inline README.md]: Also, please insert a comma after "task" :)

File: README.md [hunk 2]
```diff
@@ -182,6 +182,16 @@ open a terminal and run:
...
+
+``` bash
+# Compile CSS only
```
Comment (anselmbradford) [inline README.md]: Might say `Watch & compile CSS only.`

File: README.md [hunk 3]
```diff
@@ -182,6 +182,16 @@ open a terminal and run:
...
+$ grunt watch:css
+
+# Compile CSS & JS only
```
Comment (anselmbradford) [inline README.md]: Might say `Watch & compile CSS & JS only.`

File: README.md (hunk 1 above)
Comment (anselmbradford) [inline README.md]: With all our brains together we will figure out the English language! ;)

Comment (jimmynotjim) [inline README.md]: Me fail English? That's unpossible

Comment (anselmbradford) [inline README.md]: Irregardless I think it happened

**Comment by jimmynotjim** [comment]
Updated with suggestions

**Comment by Scotchester** [comment]
WHAT ABOUT THE COLON???? :sob:

**Comment by jimmynotjim** [comment]
@Scotchester I don't know what you're talking about ;)

**Comment by Scotchester** [comment]
You and your sneaky squashing ;)

**Pull request merged**